packages/components/addon/components/hds/form/character-count/index.hbs

```diff
@@ -0,0 +1,29 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Text::Body
+  @tag="div"
+  @size="100"
+  class={{this.classNames}}
+  id={{this.id}}
+  {{did-insert this.onInsert}}
+  {{will-destroy this.willDestroyNode}}
+  ...attributes
+  aria-live="polite"
+>
+  {{#if (has-block)}}
+    {{yield
+      (hash
+        minLength=this.minLength
+        maxLength=this.maxLength
+        currentLength=this.currentLength
+        remainingLength=this.remainingLength
```

Comment:
We should split this in two variables, one for the difference between `minLength` and `currentLength`, and one for the difference between `maxLength` and `currentLength` (names TBD, maybe `[min|max] + [Remaining/Balance/Left/Diff/Difference]`?)